### DISCUSSION TITLE
Is setting a topic/routing key in the binding implemented

### DISCUSSION THREAD

**Discussion by gripnot** [rabbitmq/messaging-topology-operator#322] [Q&A] [answered]
Is setting a topic/routing key in the binding implemented?

**Comment by ChunyiLyu** [comment, marked as answer]
Yes. It can be set in`spec.routingKey`. You can check the bindings (or any other crd) schema by `kubectl describe`, e.g.`kubectl describe crd bindings.rabbitmq.com`.

**Comment by LJSkadi** [comment]
Is it possible to bind on multiple routingKey at the same time? Or do I need to create a binding for each key separately? For example:
```
routingKey:
  - "a-routing-key"
  - "another-routing-key"
 ```
